modules/hitlnext/src/actions/handoff.js

```diff
@@ -8,17 +8,18 @@ const axios = require('axios')
  * @description Transfers control of the conversation to an agent
  * @author Botpress, Inc.
  */
-const escalate = async event => {
+const escalate = async (event, timeoutDelay) => {
```

Comment:
Can you update the JSDoc so that people don't have to guess that the `timeoutDelay` variable exists

Reply:
No, because if you use the action directly, it serves no purpose. It's only useful when called by the skill, which the user doesn't need to handle it